<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>unCompressWin</class>
 <widget class="QMainWindow" name="unCompressWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/image/splash.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>3</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/image/icon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>7</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>文件解压</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{ 
border-image: url(:/res/image/dialog_close_normal.png);
}
QPushButton:hover
{
border-image: url(:/res/image/dialog_close_hover.png);
}
QPushButton:pressed
{   
border-image: url(:/res/image/dialog_close_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_dst">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>480</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
border-radius: 5px;
border: 2px solid #ccc; 
padding: 0px 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_dst">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>解压到：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_choose_dst">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{ 
font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);	
	background-color: rgb(214, 214, 214);
border: none;
border-radius: 3px
}
QPushButton:hover
{
background-color: rgb(240, 240, 240);
}
QPushButton:pressed
{   
background-color: rgb(185, 185, 185);
}</string>
    </property>
    <property name="text">
     <string>更换目录</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>800</width>
      <height>290</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{ 
font: 12pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 178, 238);
border: none;
border-radius: 3px
}
QPushButton:hover
{
background-color: rgb(0, 191, 255);
}
QPushButton:pressed
{   
background-color: rgb(0, 160, 213);
}</string>
    </property>
    <property name="text">
     <string>立即解压</string>
    </property>
   </widget>
   <widget class="QLabel" name="hint_wait">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="show_wait">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>30</y>
      <width>512</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>bg</zorder>
   <zorder>title_bar</zorder>
   <zorder>icon</zorder>
   <zorder>title</zorder>
   <zorder>btn_close</zorder>
   <zorder>edt_dst</zorder>
   <zorder>label_dst</zorder>
   <zorder>btn_choose_dst</zorder>
   <zorder>btn_start</zorder>
   <zorder>hint_wait</zorder>
   <zorder>show_wait</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
